content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md

```diff
@@ -0,0 +1,152 @@
+
+Normally, you can follow the single step in the documentation for installing wasmCloud via Helm. However, we want to slightly customize this to enable the Kubernetes Applier to bootstrap application deployment. First, create a custom values.yaml:
+
+```yaml
+wasmcloud:
+enableApplierSupport: true
+customLabels:
+wasmcloud.dev/route-to: "true"
+```
+
+Then you can deploy via helm (with your choice of RELEASE_NAME)
+
+```shell
+helm install <RELEASE_NAME> wasmcloud/wasmcloud-host -f values.yaml
+```
+
+Finally, to verify the install, you can grab the name of the deployment:
+
+```shell
+$ kubectl get deployments
+NAME                            READY   UP-TO-DATE   AVAILABLE   AGE
+wasmcloud-test-wasmcloud-host   1/1     1            1           4m32s
+```
+
+And then forward port 4000 to localhost:
+
+```shell
+$ kubectl port-forward deployment/wasmcloud-test-wasmcloud-host 4000
+Forwarding from 127.0.0.1:4000 -> 4000
+Forwarding from [::1]:4000 -> 4000
+Handling connection for 4000
+```
+
+Then finally simply visit [localhost:4000](http://localhost:4000) in your favorite browser, and you can see the wasmCloud dashboard ready to run your first apps, all from an existing Kubernetes namespace.
+
+Right now, this is simple bootstrapped infrastructure. It runs wasmCloud hosts within the service’s namespace in Kubernetes side-by-side with the existing service. Long-term it would be more efficient to have a pool of wasmCloud hosts in the cluster to support any number of client applications.
+
+We’re also using a handy wasmCloud Actor written by Taylor Thomas at Cosmonic, the wasmCloud Kubernetes Applier, to easily deploy other actors to the wasmCloud infrastructure inside Kubernetes we just provisioned. Check it out…
+
+[The applier is hosted here](https://github.com/cosmonic/kubernetes-applier/tree/main/service-applier), although you can use `wash` commands and don’t necessarily need to clone the repository just to use the applier.
+
+Let’s set up the example `echo` actor to get things running. We’ll need to create several providers and actors, then wire them up with link definitions. First start the actors and providers [with the current latest versions](https://github.com/wasmCloud/capability-providers#first-party-capability-providers):
+
+```shell
+wash ctl start provider wasmcloud.azurecr.io/applier:0.3.0
+wash ctl start provider wasmcloud.azurecr.io/nats_messaging:0.14.5
+wash ctl start provider wasmcloud.azurecr.io/httpserver:0.16.3
+wash ctl start actor wasmcloud.azurecr.io/service_applier:0.3.0
+wash ctl start actor wasmcloud.azurecr.io/echo:0.3.4
```

Comment:
Are these the right versions? `echo` is on 0.3.5

Reply:
Yeah these are fine